content/en/docs/concepts/glossary.md

```diff
@@ -266,17 +266,17 @@ Short for [OpenTelemetry Protocol](/docs/specs/otlp/).
 ### Propagators
 
 Used to serialize and deserialize specific parts of telemetry data such as span
-context and [Baggage](#baggage) in [Spans](#span). See [more][propagators].
+context and [Baggage](#baggage) in [Spans](#span). See [Propagators].
```

Comment:
This links to the Go instrumentation page, but I think Propagators can be used in other languages? Maybe the [Propagators API](https://opentelemetry.io/docs/specs/otel/context/api-propagators/) page is better.

Reply:
Will track via:

- #6985

Reviewers, please comment in that issue.